packages/database/package.json

```diff
@@ -17,7 +17,9 @@
   "scripts": {
     "prepublishOnly": "npm run build",
     "clean": "rm -rf node_modules && rm -rf build",
-    "test": "NODE_ENV=debug jest -i tests/*.test.ts",
+    "test": "NODE_ENV=debug vitest --run tests/*.test.ts",
+    "test:watch": "NODE_ENV=debug vitest tests/*.test.ts",
+    "test:debug": "NODE_ENV=debug vitest tests/*.test.ts --inspect-brk --pool threads --poolOptions.threads.singleThread",
     "build": "rm -rf build && npx rollup -c rollup/rollup.mjs",
     "dev": "rm -rf build && npx rollup -c rollup/rollup.mjs",
     "coverage": "npm run test -- --coverage",
```

Comment:
:heavy_check_mark: `coverage` script works exactly the same, though I don't have config in place for "thesholds". It now outputs some nice pretty navigable html to see where the coverage is lacking. `.gitignore`d because it's sugar though.

The new `test:watch` and `test:debug` are suuuper nice. No more rebuild + test loop... it just builds and re-runs tests